app/Containers/AppSection/Authorization/UI/API/Requests/AttachPermissionsToUserRequest.php

```diff
@@ -35,8 +35,7 @@ class AttachPermissionsToUserRequest extends ParentRequest
     public function rules(): array
     {
         return [
-            'permissions_ids' => 'required',
-            'user_id' => 'required',
```

Comment:
there should be a validation for id for checking if the user exists.
'id' => 'required|exists:users,id,deleted_at,NULL';
This should be in all the requests.